exercises/06.1-Loop-Array/README.md

```diff
@@ -15,9 +14,8 @@ Instead of doing that:
 
 1. Print all the elements in the array. Iterate through the whole array using a loop.
 
-
 ## 💡Hint:
 
-+  Remember that to access the value of a position you have to use the index `console.log(myArray[index]);`.
++ Remember that to access the value of a position you have to use the index `console.log(myArray[index]);`.
```

Comment:
console.log (myArray [index]); the console.log() could be confusing for the student. Use just this instead: myArray[index]